src/components/AllAlerts/index.js

```diff
@@ -198,104 +193,96 @@ class AllAlerts extends Component {
-                icon={faCaretDown}
-                // size="1x"
-                sx={{
-                  position: "relative",
-                  minHeight: "150vh",
-                  paddingLeft: "5vh",
-                }}
-              /> */}
-              <option default>All</option>
+      <>
+        <Row className={styles.alertsMargin}>
+          <Col md={2}>
+            <h1 className={styles.alertsHeader}>{filteredLength} Alerts</h1>
+          </Col>
+          <Col md={10}>
+            <div className={styles.severity}>
+              <div className={styles.parentColors}>
+                <Container>
+                  <Row>
+                    <Col sm={2.5} className={styles.boldHeader}>
+                      Severity:
+                    </Col>
+                    <div className={styles.flexCol}>
+                      <Col md={11} className={styles.upperRight}>
+                        <Row>
+                          <div className={styles.parentAlerts}>
+                            <div className={styles.red}></div>
+                            <div className={styles.sevParagraph}>
+                              {this.state.highCount} High
+                            </div>
+                          </div>
 
-              {sliced.map((tag, i) => {
-                return (
-                  <option value={tag} key={i}>
-                    {tag}
-                  </option>
-                )
-              })}
-            </select>
-          </div>
-          <div className={styles.severity}>
-            <div className={styles.parentColors}>
-              <Container>
-                <Row>
-                  <Col sm={2.5} className={styles.boldHeader}>
-                    Severity:
-                  </Col>
-                  <div className={styles.flexCol}>
-                    <Col md={11} className={styles.upperRight}>
-                      <Row>
-                        <Col sm={4} className={styles.flexCol}>
-                          <div
-                            className={classNames(styles.box, styles.red)}
-                          ></div>
-                          <p className={styles.sevParagraph}>{this.state.highCount} High</p>
-                        </Col>
+                          <div className={styles.parentAlerts}>
+                            <div className={styles.orange}></div>
+                            <div className={styles.sevParagraph}>
+                              {this.state.midCount} Mid
+                            </div>
+                          </div>
 
-                        <Col sm={4} className={styles.flexCol}>
-                          <div
-                            className={classNames(styles.box, styles.orange)}
-                          ></div>
-                          <p className={styles.sevParagraph}>{this.state.midCount} Mid</p>
-                        </Col>
+                          <div className={styles.yellow}></div>
```

Comment:
This isn't inside a div with `parentAlerts` like the others - I think it should be for consistency.

Reply:
that's been fixed